### PR TITLE
Publish v2023.05.1. [release]

### PR DESCRIPTION
**Pull request by ryanbourdais**

### CODE REVIEW

File: 2023.05/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# vim:set ft=dockerfile:
+
+# Do not edit individual Dockerfiles manually. Instead, please make changes to the Dockerfile.template, which will be used by the build script to generate Dockerfiles.
+
+# By policy, the base image tag should be a quarterly tag unless there's a
+# specific reason to use a different one. This means January, April, July, or
+# October.
+
+FROM cimg/base:2023.03
```
Comment (JalexChen) [inline 2023.05/Dockerfile:9]: this should be 2023.04 to match with the quarterly cimg/base image

**Review by JalexChen — CHANGES_REQUESTED** [review]

File: 2023.05/Dockerfile (hunk 1 above)
Comment (ryanbourdais) [reply 2023.05/Dockerfile:9]: good catch, I don't know why I made it March

Comment (JalexChen) [reply 2023.05/Dockerfile:9]: @ryanbourdais you still need to generate the dockerfile so it's reflected in `2023.05/Dockerfile`

Comment (ryanbourdais) [reply 2023.05/Dockerfile]: @JalexChen good catch, totally forgot that. Should be good now

File: 2023.05/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+ENV M2_HOME /usr/local/apache-maven
+ENV MAVEN_OPTS -Xmx2048m
+ENV PATH $M2_HOME/bin:$PATH
+# Set JAVA_HOME (and related) environment variable. This will be set to our
+# default Java version of 17 but the user would need to reset it when changing
+# JAVA versions.
+ENV JAVA_HOME=/usr/lib/jvm/java-17-openjdk-amd64
+ENV JDK_HOME=${JAVA_HOME}
+ENV JRE_HOME=${JDK_HOME}
+ENV MAVEN_VERSION=3.9.0
```
Comment (JalexChen) [inline 2023.05/Dockerfile:34]: in your latest commit, maven version went from 3.9.1 to 3.9.0

**Review by JalexChen — CHANGES_REQUESTED** [review]

**Review by JalexChen — APPROVED** [review]

**Pull request merged**